UnitTestBot/UTBotJava
# Reorder components in UI

# Description

We should better place all components in UI in the following order:
- test framework
- mock framework
- use static mocking?
- use parametrization?

See internal discussions in UtBot Java Settings chat.

## Type of Change

- Refactoring (typos and non-functional changes) 

# How Has This Been Tested?

## Manual Scenario 

Verify the order of mentioned controls and their functionality.
